Could not type "/sw hud"

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Do something...
2. After which...
3. And then...
4. This error appears...

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Versions (please complete the following information):**
 - Towny: 
 - SiegeWar: 
 - Spigot/Paper:

**Files**
Startup log:
SiegeWar config:
Townyperms.yml:

[12:05:15 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'sw' in plugin SiegeWar v2.6.0
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[folia-api-1.19.4-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[folia-api-1.19.4-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R3.CraftServer.dispatchCommand(CraftServer.java:1017) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at org.bukkit.craftbukkit.v1_19_R3.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[folia-1.19.4.jar:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:322) ~[?:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:306) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.performChatCommand(ServerGamePacketListenerImpl.java:2310) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$16(ServerGamePacketListenerImpl.java:2270) ~[?:?]
        at io.papermc.paper.threadedregions.EntityScheduler.executeTick(EntityScheduler.java:173) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1527) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at io.papermc.paper.threadedregions.TickRegions$ConcreteRegionTickHandle.tickRegion(TickRegions.java:360) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at io.papermc.paper.threadedregions.TickRegionScheduler$RegionScheduleHandle.runTick(TickRegionScheduler.java:385) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool$TickThreadRunner.run(SchedulerThreadPool.java:525) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at java.lang.Thread.run(Unknown Source) ~[?:?]
Caused by: java.lang.UnsupportedOperationException
        at org.bukkit.craftbukkit.v1_19_R3.scoreboard.CraftScoreboardManager.getNewScoreboard(CraftScoreboardManager.java:45) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at org.bukkit.craftbukkit.v1_19_R3.scoreboard.CraftScoreboardManager.getNewScoreboard(CraftScoreboardManager.java:25) ~[folia-1.19.4.jar:git-Folia-"ca3b7ad"]
        at com.gmail.goosius.siegewar.hud.SiegeWarHud.toggleOn(SiegeWarHud.java:46) ~[SiegeWar-2.6.0.jar:?]
        at com.gmail.goosius.siegewar.hud.SiegeHUDManager.toggleWarHud(SiegeHUDManager.java:26) ~[SiegeWar-2.6.0.jar:?]
        at com.gmail.goosius.siegewar.command.SiegeWarCommand.parseSiegeWarHudCommand(SiegeWarCommand.java:218) ~[SiegeWar-2.6.0.jar:?]
        at com.gmail.goosius.siegewar.command.SiegeWarCommand.parseSiegeWarCommand(SiegeWarCommand.java:136) ~[SiegeWar-2.6.0.jar:?]
        at com.gmail.goosius.siegewar.command.SiegeWarCommand.onCommand(SiegeWarCommand.java:113) ~[SiegeWar-2.6.0.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[folia-api-1.19.4-R0.1-SNAPSHOT.jar:?]
        ... 14 more
[12:05:26 ERROR]: Could not pass event NewShortTimeEvent to SiegeWar v2.6.0

--- a/src/main/java/com/gmail/goosius/siegewar/SiegeWar.java
+++ b/src/main/java/com/gmail/goosius/siegewar/SiegeWar.java
@@ -237,7 +237,7 @@ public class SiegeWar extends JavaPlugin {
 		return null; // Doesn't matter because SiegeWar will not pass the onEnable phase.
 	}
 
-	private static boolean isFoliaClassPresent() {
+	public static boolean isFoliaClassPresent() {
 		try {
 			Class.forName("io.papermc.paper.threadedregions.RegionizedServer");
 			return true;

--- a/src/main/java/com/gmail/goosius/siegewar/command/SiegeWarCommand.java
+++ b/src/main/java/com/gmail/goosius/siegewar/command/SiegeWarCommand.java
@@ -204,6 +204,9 @@ public class SiegeWarCommand implements CommandExecutor, TabCompleter {
 
 	private void parseSiegeWarHudCommand(Player player, String[] args) {
 		try {
+			if(SiegeWar.isFoliaClassPresent()) {
+				throw new TownyException(Translatable.of("msg_err_hud_not_available_in_folia"));
+			}
 			if (args.length == 0) {
 				TownyMessaging.sendMessage(player, ChatTools.formatTitle("/siegewar hud"));
 				TownyMessaging.sendMessage(player, ChatTools.formatCommand("Eg", "/sw hud", "[town]", ""));

--- a/src/main/resources/lang/en-US.yml
+++ b/src/main/resources/lang/en-US.yml
@@ -182,6 +182,7 @@ msg_war_siege_battle_session_break_cannot_get_banner_control: "&cBanner control 
 hud_siege_balance: 'Siege Balance: '
 hud_warchest: 'War Chest: '
 hud_battle_time_remaining: 'BAT Time Left: '
+msg_err_hud_not_available_in_folia: "&cThe HUD is not available in Folia."
 
 #Dynmap
 dynmap_siege_battle_time_left: 'Battle Time Left: %s'

